loader/src/jmh/java/cpw/mods/modlauncher/benchmarks/TransformBenchmark.java

```diff
@@ -39,72 +30,59 @@
 import org.openjdk.jmh.annotations.Setup;
 import org.openjdk.jmh.annotations.State;
 import org.openjdk.jmh.annotations.TearDown;
-import org.powermock.reflect.Whitebox;
 
 @State(Scope.Benchmark)
 public class TransformBenchmark {
     public volatile ClassTransformer classTransformer;
-    static Method transform;
+    public volatile TransformerAuditTrail auditTrail;
     byte[] classBytes;
 
     @Setup
     public void setup() throws Exception {
         final TransformStore transformStore = new TransformStore();
-        final ModuleLayerHandler layerHandler = Whitebox.invokeConstructor(ModuleLayerHandler.class);
-        final LaunchPluginHandler lph = new LaunchPluginHandler(layerHandler);
-        classTransformer = uncheck(() -> Whitebox.invokeConstructor(ClassTransformer.class, new Class[] { transformStore.getClass(), lph.getClass(), TransformingClassLoader.class }, new Object[] { transformStore, lph, null }));
-        transform = uncheck(() -> classTransformer.getClass().getDeclaredMethod("transform", byte[].class, String.class, String.class));
-        transform.setAccessible(true);
-        Map<String, ILaunchPluginService> plugins = Whitebox.getInternalState(lph, "plugins");
-        try (InputStream is = getClass().getClassLoader().getResourceAsStream("cpw/mods/modlauncher/testjar/TestClass.class")) {
-            ByteArrayOutputStream bos = new ByteArrayOutputStream();
-            byte[] buf = new byte[2048];
-            while (is.read(buf) >= 0) {
-                bos.write(buf);
-            }
-            classBytes = bos.toByteArray();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-        plugins.put("dummy1", new ILaunchPluginService() {
-            @Override
-            public String name() {
-                return "dummy1";
-            }
+        final LaunchPluginHandler lph = new LaunchPluginHandler(Stream.of(
```

Comment:
I don't know how much you want to clean up or not here. Cause we should potentially rename `final LaunchPluginHandler lph` to `var launchPluginHandler`.